src/include/server_limits.h

```diff
@@ -42,9 +42,7 @@
 #ifdef	__cplusplus
 extern "C" {
 #endif
-#ifdef WIN32
-#include "win.h"
-#endif
+#include <pbs_config.h>
```

Comment:
I only see a few places where removal of "win.h" (and sometimes winsock.h") is replaced with #include <pbs_config.h>. What determines this? Saw this also in src/lib/Libutil/entlim.c and src/tools/pbs_sleep.c, but not in:
src/resmom/job_recov_fs.c,
src/resmom/requests.c,
src/resmom/stage_func.c,
src/server/job_func.c
src/server/req_quejob.c

Reply:
Hi @bayucan Thanks for taking a look in this PR.
Scope of this PR is to move away from any specific header file inclusion depnending on WIN32.
For Windows both "win.h" and "winsock.h" is part of pbs_config.h now.
W didn't add that on those file, because its already there.

Reply:
Ok.